raiden/storage/serialization/fields.py

```diff
@@ -11,20 +11,20 @@
 
 
 class IntegerToStringField(marshmallow.fields.Field):
-    def _serialize(self, value: int, attr: Any, obj: Any) -> str:
+    def _serialize(self, value: int, attr: Any, obj: Any, **kwargs) -> str:
```

Comment:
Where does the need for the `kwargs` everywhere come from?

Reply:
Good question, for the hooks it's mentioned in the changelog: https://github.com/marshmallow-code/marshmallow/blob/dev/CHANGELOG.rst#300rc7-2019-06-15

But for these I got test failures pointing at our custom fields, and then pyCharm showed that the signatures didn't match the base class.

Reply:
The `**kwargs` are also in the docs (https://marshmallow.readthedocs.io/en/3.0/custom_fields.html#creating-a-field-class), so apparently that's the way to do it.